ui-tests/src/test/java/io/syndesis/qe/pages/connections/edit/ConnectionEditPage.java

```diff
@@ -8,21 +8,26 @@
 import com.codeborne.selenide.SelenideElement;
 
 import io.syndesis.qe.pages.SyndesisPageObject;
+import lombok.Getter;
 import lombok.extern.slf4j.Slf4j;
 
 /**
  * Created by sveres on 11/12/17.
  */
+@Getter
 @Slf4j
-public class ConnectionCreatePage extends SyndesisPageObject {
+public class ConnectionEditPage extends SyndesisPageObject {
```

Comment:
I'm still for preserving ConnectionCreatePage name, but we can discuss it in office.

Reply:
@lvydra I prefer ConnectionEditPage, ok. let's discuss in the office.

Reply:
@Stefan365 For me the main problem is that no such page exists.

Reply:
I'd say, lets stick with former names and merge this quiet big PR. Classname are easily refactored in the future.